server/views/components/actions-card/actionsCard.test.ts

```diff
@@ -9,28 +9,129 @@ nunjucks.configure([
   __dirname,
 ])
 describe('Tests for actions card component', () => {
-  it('Should display a link for each action', () => {
-    const actions: Action[] = [
-      { title: 'Do a thing', href: '/thing' },
-      { title: 'Do another thing', href: '/another-thing' },
-    ]
-    const content = nunjucks.render('test.njk', { actions })
-    const $ = cheerio.load(content)
-    const actionListItems = $('.actions-list-list').find('li')
-    expect(actionListItems.length).toStrictEqual(2)
-    expect($(actionListItems[0]).find('a').text()).toStrictEqual('Do a thing')
-    expect($(actionListItems[0]).find('a').attr('href')).toStrictEqual('/thing')
-    expect($(actionListItems[1]).find('a').text()).toStrictEqual('Do another thing')
-    expect($(actionListItems[1]).find('a').attr('href')).toStrictEqual('/another-thing')
+  describe('Core action card component', () => {
```

Comment:
I've not deleted the existing tests; I've added lots of new ones (based on permutations of the `id` and `attributes` and `render-if` properties); and also grouped them into nested `describe` blocks (1 for the action card component itself, and one for the action items)